packages/zk-transfer-web/src/util.ts

```diff
@@ -17,11 +17,12 @@ export const generateScreenName = () => {
 };
 
 export const getZkTokenName = (token: string | number) => {
```

Comment:
`token` here is better called `tokenDeployName`, which elsewhere in the code we understand to mean a name that looks like `<SmartContractName>-deploy<deployId>`.

Reply:
All this method name should be called `getZkTradeSymbol`, it's more accurate. Thanks

Reply:
Done!